inch.go

```diff
@@ -313,21 +321,31 @@ func (s *Simulator) generateBatches() <-chan []byte {
 			fields = append(fields, []byte(fmt.Sprintf("v%d=1%s", i, delim))...)
```

Comment:
I know this line wasn't touched in this PR but the byte slice conversion is unnecessary.